### PR TITLE
Feature/add map themes

### PR DESCRIPTION
**Pull request by virginia-d90**
### Description
Clusters the filtered map points to allow for better visibility. The clusters also have zoom functionality when clicked.

### Files altered
Map.js
map.css
Store.js

### CODE REVIEW

File: src/components/pages/Map/map.css [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+.cluster-marker {
+  color: #fff;
+  background: #1978c8;
+  border-radius: 50%;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
 /* .marker-btn:hover {
```
Comment (Khalil-Foulks) [inline src/components/pages/Map/map.css:15]: This commented out code should be taken out.

**Review by Khalil-Foulks — APPROVED** [review]
Looked good, there was commented out code in map.css that should be removed.

**Review by mrockingham — COMMENTED** [review]
Looks good and working

File: src/components/pages/Map/map.css (hunk 1 above)
Comment (virginia-d90) [reply src/components/pages/Map/map.css:15]: changes committed

**Pull request merged**